src/parse.ts

```diff
@@ -930,6 +955,10 @@ function parseMediaPlaylist(lines, params) {
             discontinuitySequence,
             params,
         );
+
+        // Apply user-specified segment transformers
+        segment = transformSegment(segment, playlist, params);
```

Comment:
Should this go _after_ the `if` on 962?

Reply:
I was allowing for the transformer to remove segments. Also, `parseSegment` always returns a new Segment, so the original check seems unnecessary